pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -635,6 +641,9 @@ public void setOfflineSegmentIntervalCheckerInitialDelayInSeconds(long initialDe
   }
 
   public long getPeriodicTaskInitialDelayInSeconds() {
+    if (containsKey(ControllerPeriodicTasksConf.SEGMENT_LEVEL_VALIDATION_INTERVAL_IN_SECONDS)) {
+      return getLong(ControllerPeriodicTasksConf.SEGMENT_LEVEL_VALIDATION_INTERVAL_IN_SECONDS);
+    }
```

Comment:
getLong(key, defaultValue) should reduce this to 1 line

Reply:
Disagree. We don't want users configuring this value, or provide a default if they don't. We want to choose random delay all the time except for tests.